pages/404.js

```diff
@@ -0,0 +1,18 @@
+import Button from "../components/Button";
+import TextSection from "../components/TextSection.jsx";
+
+const errorPage404 = () => {
```

Comment:
nit: Should we capitalize the e just so it's consistent with the rest of the components? So it will be `ErrorPage404`?